gnocchi/rest/__init__.py

```diff
@@ -665,6 +665,9 @@ def post(self):
         except indexer.NoSuchResource as e:
             abort(404, e)
 
+        return pecan.request.indexer.get_resource(
```

Comment:
I agree. 

On my first commit before rebasing I was only returning the newly created metrics. But @sileht mentioned something about returning the whole resource's description, which would return the new metrics as well.

Waiting on more information to know how to proceed with this.